liquidator/test/Liquidator.js

```diff
@@ -171,4 +171,17 @@ contract("Liquidator.js", function(accounts) {
     assert.deepStrictEqual(await emp.getLiquidations(sponsor3), []);
     assert.equal((await emp.getCollateral(sponsor3)).rawValue, toWei("175"));
   });
+
+  it.only("set up positions", async function() {
+    console.log("seeding emp @", emp.address);
+    for (let i = 1; i < 6; i++) {
```

Comment:
Why was this test added here? Does this test the slack integration?

Reply:
same as before, was using it to set up a testing environment. should not have been committed & I've removed it.